src/main/webapp/services/CommentService.js

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios';
+import { getIntegrationBaseUrl } from 'symphony-integration-commons';
+import BaseService from './BaseService';
+
+const unexpectedErrorDialog = require('../templates/unexpectedErrorDialog.hbs');
+const forbiddenDialog = require('../templates/forbiddenDialog.hbs');
+const notFoundDialog = require('../templates/issueNotFoundDialog.hbs');
+const errorDialog = require('../templates/errorDialog.hbs');
+const commentDialog = require('../templates/commentDialog.hbs');
+
+export default class CommentService extends BaseService {
+  constructor(serviceName) {
+    super(serviceName);
+    this.baseUrl = getIntegrationBaseUrl();
+    this.comment = '';
+  }
+
+  openCommentDialog(data, serviceName) {
+    const dialogsService = SYMPHONY.services.subscribe('dialogs');
+
+    const template = commentDialog();
+
+    const commentData = {
+      comment: {
+        service: serviceName,
+      },
+      commentIssue: {
+        service: serviceName,
+        label: 'OK',
+        data: {
+          entity: data.entity,
+          type: 'commentIssue',
+        },
+      },
+      closeCommentDialog: {
+        service: serviceName,
+        label: 'Cancel',
+        data: {
+          entity: data.entity,
+          type: 'closeCommentDialog',
+        },
+      },
+    };
+
+    dialogsService.show('commentIssue', serviceName, template, commentData, {});
+  }
+
+  commentIssue(url, issueKey) {
+    const apiUrl = `${this.baseUrl}/v1/jira/rest/api/issue/${issueKey}/comment`;
```

Comment:
The axios calls can be put in a separated file, as FE-Commons. This way the urls are kept in one place.
Wdyt?